### ISSUE TITLE
[History] Remove Duplicate "Hero Moved" Entries

### ISSUE THREAD

**Issue by silent-arbiter** [Ryuasd/hQuestMaster#37]
In the "History" tab, if the hero moves, I think "hero Moved" should show up once rather than multiple times in a row. Having the hero move, open a door, and move again to me seems like it should count as 1 movement. Also, "hero Moved" is not capitalized in the history panel. Basically, if the previous action was "Hero Moved," don't add more to the History Log for each hero movement to prevent excess clutter (since they don't say which hero moved, the duplicate entries seem burdensome). Maybe it could say Player Movement, Monster Movement, etc.

Also: "hero Moved" is not capitalized in the game log

**Comment by Ryuasd** [comment]
You are right, but the History feature is still a work in progress, in fact in the next release will be temporary removed.
